hack/charts/installer/rolebased/README.md

```diff
@@ -1,5 +1,49 @@
-things that need to be converted to helm:
-* AWS_DEFAULT_SAGEMAKER_ENDPOINT (TODO)
+# Amazon SageMaker Operator for Kubernetes Role-Based Installer
 
-Notes:
-* `kustomize build config/helm` -> has names that are actually variable references. We then replace those using helm.
+Amazon SageMaker Operator for Kubernetes Helm charts for role-based installation
+
+## Prerequisites
+
+* Kubernetes >= 1.13
+
+## Installing the Chart
+
+Clone the existing repository:
+
+```bash
+git clone https://github.com/aws/amazon-sagemaker-operator-for-k8s.git
+```
+
+Navigate to the helm chart directory and edit the values in the configuration file:
+
+```bash
+cd amazon-sagemaker-operator-for-k8s/hack/charts/installer
+vim rolebased/values.yaml
+```
+
+The [configuration](#configuration) section below lists the parameters that can be configured for installation.
+
+Install the operator using the following command:
+
+```bash
+kubectl create namespace sagemaker-k8s-operator-system
```

Comment:
Is this necessary? I thought that the helm YAML had the namespace specification already.

Reply:
If you do it defaults the {{ .Release.Namespace }} to `default`, sadly.

Reply:
I don't understand why this is bad, maybe I'll come over.

Reply:
This should be optional and our installer.yaml should have  `sagemaker-k8s-operator-system` as default namespace.